src/components/ContactsList/__snapshots__/ContactsListDataLoader.spec.jsx.snap

```diff
@@ -41,33 +41,30 @@ exports[`ContactsListOrSpinner should display the contact List 1`] = `
             >
               <div
                 className="styles__c-avatar___PpDI- styles__c-avatar--small___1Y_Pv"
+                style={
+                  Object {
+                    "backgroundColor": "var(--puertoRico)",
+                    "color": "white",
+                  }
+                }
               >
                 
-                
-                <svg
-                  className="styles__icon___23x3R"
-                  height="16"
-                  style={Object {}}
-                  width="16"
```

Comment:
why this removal?

Reply:
Because for those scenarios (no name but email address or cozy url) there is no longer default avatar but avatar with initials